.github/workflows/pr_testing.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
```

Comment:
@daniel-thom can you please do these changes to the same file in PSY whenever you do your PR.